cypress/e2e/SpamCheckAPI/documentation.cy.js

```diff
@@ -0,0 +1,97 @@
+import { SPAM_CHECK_API_BASE_URL } from '../../resources/data';
+
+describe('Spam Check API Documentation page', () => {
+    before(() => {
+      cy.visit(SPAM_CHECK_API_BASE_URL);
+    });
+
+    it('1. Test to "Accept Cookies" if not already accepted', () => {
+        cy.AcceptCookies();
+    });   
+
+    it('2. Test to check "Documentation" tab visibility', () => {
+        cy.componentVisiblityCheck('.d-md-flex > .d-none');
+    });
+
+    it('3. Test to click "Documentation" tab', () => {
+        cy.intercept('GET', '/marketplace/spamchecker-api/tabs/api_docs').as('apiRequest');
+        cy.get('.d-md-flex > .d-none').click();
+        cy.wait('@apiRequest');
+    });
+
+    it('4. Test to check "Contents:" span visibility and content', () => {
+        cy.componentVisiblityCheck('span.text-muted.font-weight-bold', 'Contents:');
+    });
+
+    it('5. Test to check "Spam Check API Reference" title is present', () => {
+        cy.checkTextVisibility('Spam Check API Reference');
+    });
+
+    it('6. Test to check "Just Getting Started?" title is present', () => {
+        cy.checkTextVisibility('Just Getting Started?');
+    });
+
+    it('7. Test to check "development quickstart guide" link visibility and click behavior', () => {
+        cy.componentVisiblityCheck('a[href="/docs/article/getting-started"]', 'development quickstart guide.');
```

Comment:
Can ypu please create a variable for this url and reuse it?

Reply:
Instead of the url i am getting the text which has hyperlink and clicking it   cy.contains('a', 'development quickstart guide').click();